es-client.yaml

```diff
@@ -16,20 +16,16 @@ spec:
       initContainers:
       - name: init-sysctl
         image: busybox
-        imagePullPolicy: IfNotPresent
-        command: ["sysctl", "-w", "vm.max_map_count=262144"]
+        imagePullPolicy: Always
+        command:
+        - sysctl
+        - -w
+        - vm.max_map_count=262144
         securityContext:
           privileged: true
       containers:
       - name: es-client
-        securityContext:
-          privileged: false
-          capabilities:
-            add:
-              - IPC_LOCK
-              - SYS_RESOURCE
         image: quay.io/pires/docker-elasticsearch-kubernetes:6.1.0
-        imagePullPolicy: Always
```

Comment:
Why remove this?

Reply:
the image is pinned to a specific tag. The "always" pull policy slows down startup time of the pods.

Reply:
Why then use the same policy for init-containers?

Reply:
the image is not pinned.

Reply:
Interesting. Maybe we should pin it?